packages/vite/src/node/http.ts

```diff
@@ -187,10 +187,8 @@ export async function httpServerStart(
 ): Promise<number> {
   let { port, strictPort, host, logger } = serverOptions
 
-  // This could be removed when Vite only supports Node 17+ because verbatim=true is default
-  // https://github.com/nodejs/node/pull/39987
   if (host === 'localhost') {
-    const addr = await dns.lookup('localhost', { verbatim: true })
+    const addr = await dns.lookup('localhost')
     host = addr.address
   }
```

Comment:
I think we could completely remove this if block.
host will be resolved internally in `httpServer.listen`
https://github.com/nodejs/node/blob/177064d0aee4a331eef93e3667963c02f0b94333/lib/net.js#L1504-L1505

Reply:
Updated!